scene/main/node.cpp

```diff
@@ -1269,6 +1269,7 @@ void Node::remove_child(Node *p_child) {
 		}
 	}
 
+	ERR_EXPLAIN(String("Cannot remove child node ") + p_child->to_string() + String(" as it is not in our list of children"));
```

Comment:
Why are you constructing strings from strings? Why not simply:

    ERR_EXPLAIN("Cannot remove child node " + p_child->to_string() + " as it is not in our list of children.");

Reply:
It should work without the explicit casting indeed.

Reply:
Because I'm unfamiliar with Godot's code and I come from Qt where that line wouldn't compile.